### ISSUE TITLE
Show in sidebar

### ISSUE THREAD

**Issue by Merangle** [rfxcom2mqtt/hassio-addons#4]
Very useful add-on.
Do you think it could be possible to add the UI in the sidebar like Zigbee2mqtt ?
It will be easier when in remote as I would avoid to open additional port
Many thanks

**Comment by sguernion** [comment]
yes, this is possible, but there is currently a problem with the ingress configuration for rfxcom2mqtt API calls.

you need to configure an additional "public_api" parameter in the addons configuration to get around the problem.

for example :
http://localhsot:8123/<addon_uid>/ingress
